modules/Kconfig.stm32

```diff
@@ -656,7 +691,7 @@ config USE_STM32_LL_DELAYBLOCK
 config USE_STM32_LL_DLYB
 	bool
 	help
-	  Enable STM32Cube DelayBlock (DELAYBLOCK) LL module driver (stm32U5)
+	  Enable STM32Cube DelayBlock (DELAYBLOCK) LL module driver (stm32U5 or STM32N6)
```

Comment:
```patch
 config USE_STM32_LL_DLYB
 	bool		bool
+	depends on	SOC_SERIES_STM32U5X || SOC_SERIES_STM32N6X
 	help		help
 	  Enable STM32Cube DelayBlock (DELAYBLOCK) LL module driver (stm32U5 or STM32N6)
```
?

Reply:
For now we didn't specify these dependencies. If we do, we'd need to review all `USE_STM32_` symbols and I don't think it would scale